src/Photos.vue

```diff
@@ -170,7 +170,7 @@ export default {
 				navigator.serviceWorker.register(generateUrl('/apps/photos/service-worker.js', {}, {
 					noRewrite: true,
 				}), {
-					scope: '/',
+					scope: generateUrl('/apps/photos'),
```

Comment:
However assets might be loaded through a different route so not sure if this would really work. Thoughts?

Reply:
Previews are now loaded from `/apps/photos/api/v1/preview` so it makes sense

Reply:
From the doc: the `scope` property is the "range of URLs a service worker can control". So it makes sense.

https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#parameters

Reply:
all right, cool!

Reply:
Initially we had an impact on all previews, but now that we are using a custom route for previews that only applies on photos, I guess that make sense :)

Reply:
Great! Is this custom route only for NC25?

Reply:
yep

Reply:
https://github.com/nextcloud/photos/blob/2cad0f85b7ccb4e98735823a8ca8b014d486029c/appinfo/routes.php#L123-L129